static/docs/user-guide/experiments/index.md

```diff
@@ -0,0 +1,22 @@
+# Managing Experiments
+
+Data science process is inherently iterative and R&D like. Data scientist may
+try many different approaches, different hyper-parameter values, and "fail" many
+times before the required level of a metric is achieved. Even failed experiments
+can be a useful source of information in ML.
+
+DVC makes it easy to iterate on your project, providing ways to try different
+ideas, keep track of them, switch back and forth, compare their performance
+through metrics, and find the best experiment. It stores all the context
+necessary to reproduce easily and efficiently an experiment: data, pipeline
+stages, parameters, models, etc. That way, someone else (or you yourself 3
+months from now) can check out and inspect all the details of an experiment.
+
+You can use several ways to manage experiments, which are described on this
+section. Which one is more suitable for you depends on your preferences and also
+on the kind and complexity of your project.
+
+- [How to Manage Experiments by Tags](/doc/user-guide/experiments/tags)
+- [How to Manage Experiments by Branches](/doc/user-guide/experiments/branches)
+- [How to Manage Experiments by Directories](/doc/user-guide/experiments/dirs)
+- [How to Manage Experiments by Several Methods](/doc/user-guide/experiments/mixed)
```

Comment:
by a Mix of Methods? Several methods sounds strange a bit

Reply:
I don't actually mind, choose the wording that seems right to you.

Reply:
@jorgeorpinel what's your take on this and on the one https://github.com/iterative/dvc.org/pull/828#discussion_r354483893 here? what the best way to word it?

Reply:
Maybe ** How to Manage Experiments with a Mix of Methods**
or use the word "Combination", as in that doc's intro.